src/routes/+page.md

```diff
@@ -17,13 +17,14 @@ Links:
 
 ## Installation
 
-To configure your project, add these 2 packages:
+To configure your project, add these 3 packages:
 
 ```bash
-pnpm install -D stripe svelte-stripe
+pnpm install -D stripe @stripe/stripe-js svelte-stripe
 ```
 
-- [stripe](https://npmjs.org/package/stripe) is the official server-side version of Stripe.
+- [stripe](https://npmjs.org/package/stripe) is Stripe's official server-side library.
+- [@stripe/stripe-js](https://www.npmjs.com/package/@stripe/stripe-js) is Stripe's official client-side library, which always loads the latest version of [Stripe.js](https://stripe.com/docs/stripe-js) to ensure [PCI compliance](https://stripe.com/docs/security/guide#validating-pci-compliance).
```

Comment:
I would shorten it to "is Stripe's official client-side library."

My reasoning is to keep the docs as short as possible, so people don't have to read too much to get running.